### PR TITLE
gpt config

### PR DESCRIPTION
**Pull request by hhpang**
added gpt config

### CODE REVIEW

File: config_templates/gretel/synthetics/README.md [hunk 1]
```diff
@@ -15,3 +15,5 @@ Templates can be downloaded and modified for use with Gretel synthetics, or impo
  |`high-dimensionality`| Use for datasets with more than 20 columns and/or 50,000 rows. Works well for largely numeric data. Avoid if dataset contains free text fields.  |
  |`high-accuracy`| Useful for optimizing data for downstream ML tasks, at possible cost of higher compute. |
  |`low-record-count`| For datasets that have fewer than 1000 rows and/or 4 columns, and a mix of categorical, numerical, and continuous values.  |
+ |`time-series`| Specialized model for time-series datasets. Modify this config with values that fit your dataset.  |
+ |`natural-language-gtp`| Useful for single-column natural language datasets such as reviews, tweets, and conversations. Dataset must be single-column. |
```
Comment (johntmyers) [inline config_templates/gretel/synthetics/README.md:19]: `gtp`, should this be `gpt`?

Comment (hhpang) [reply config_templates/gretel/synthetics/README.md:19]: oh no, yes, thx for catching. Also curious if @sami-t you'd agree on the time series description

Comment (hhpang) [reply config_templates/gretel/synthetics/README.md]: fixed typo

File: config_templates/gretel/synthetics/natural-language-gpt.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+models:
+  - gpt_x:
+      data_source: "__tmp__"
+      pretrained_model: 'EleutherAI/gpt-neo-125M'
+      batch_size: 4
+      epochs: 3
+      weight_decay: 0.1
+      warmup_steps: 100
+      lr_scheduler: "cosine"
+      learning_rate: 5e-6
```
Comment (yamini) [inline config_templates/gretel/synthetics/natural-language-gpt.yml:17]: minor: Newline needed at end of yaml file

Comment (hhpang) [reply config_templates/gretel/synthetics/natural-language-gpt.yml]: ah, added 👍

**Review by yamini — APPROVED** [review]
lgtm!! 🚢

File: config_templates/gretel/synthetics/README.md (hunk 1 above)
Comment (hhpang) [reply config_templates/gretel/synthetics/README.md]: edited name after convo with @sami-t. Thanks!

**Pull request merged**